projects/1.20/assets/tropicraft/tropicraft/lang/zh_cn.json

```
@@ -416,7 +416,7 @@
   "item.tropicraft.orangeade": "橙汁",
   "item.tropicraft.pab_shell": "Pab贝壳",
   "item.tropicraft.pianguas": "扇贝",
-  "item.tropicraft.pina_colada": "凤梨可乐达",
+  "item.tropicraft.pina_colada": "菠萝可乐达",
```

https://zh.wikipedia.org/wiki/%E9%B3%B3%E6%A2%A8%E5%8F%AF%E6%A8%82%E9%81%94
参考如上，我认为这个更改是不合适的
“凤梨可乐达”是更加广泛使用的名称

---

`凤梨可乐达`、`凤梨汁`、`菠萝可乐达`、`菠萝汁` 都是农产品菠萝/凤梨的衍生词，应当和农产品名称保持一致。

根据指示精神，需要保持统一，无论选择菠萝还是凤梨，与农产品名称应该统一。维基百科是不可信来源，这里决定统一方向后，可以按需修改维基百科。

---

群内目前讨论争执不一，要不就改成椰林飘香

---

是广为流传的中文另起的名字，个人没有意见

```suggestion
  "item.tropicraft.pina_colada": "椰林飘香",
```